main.py

```diff
@@ -1,10 +1,40 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as f:
+        players_info = json.load(f)
+
+    for pl in players_info.keys():
+        race, created = Race.objects.get_or_create(
+            name=players_info[pl]["race"]["name"],
+            description=players_info[pl]["race"]["description"]
+        )
+
+        if created and players_info[pl]["race"]["skills"]:
```

Comment:
Ensure that `players_info[pl]["race"]["skills"]` is a list before iterating over it to prevent potential runtime errors if the data structure is not as expected.